src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -2630,26 +2630,26 @@ namespace ReportSizingManager {
                 } else if (SizingType == SystemAirflowSizing) {
                     if (HRFlowSizingFlag) { // HX sizing
                         if (CurOASysNum) {
-                            if (FinalSysSizing(CurSysNum).DesOutAirVolFlow > 0.0) {
-                                AutosizeDes = FinalSysSizing(CurSysNum).DesOutAirVolFlow;
-                            } else {
-                                // ELSE size to supply air duct flow rate
-                                {
-                                    auto const SELECT_CASE_var(CurDuctType);
-                                    if (SELECT_CASE_var == Main) {
-                                        AutosizeDes = FinalSysSizing(CurSysNum).DesMainVolFlow;
-                                    } else if (SELECT_CASE_var == Cooling) {
-                                        AutosizeDes = FinalSysSizing(CurSysNum).DesCoolVolFlow;
-                                    } else if (SELECT_CASE_var == Heating) {
-                                        AutosizeDes = FinalSysSizing(CurSysNum).DesHeatVolFlow;
-                                    } else if (SELECT_CASE_var == Other) {
-                                        AutosizeDes = FinalSysSizing(CurSysNum).DesMainVolFlow;
-                                    } else {
-                                        AutosizeDes = FinalSysSizing(CurSysNum).DesMainVolFlow;
-                                    }
+                            // size to supply air duct flow rate
+                            {
+                                auto const SELECT_CASE_var(CurDuctType);
+                                if (SELECT_CASE_var == Main) {
+                                    AutosizeDes = FinalSysSizing(CurSysNum).DesMainVolFlow;
+                                } else if (SELECT_CASE_var == Cooling) {
+                                    AutosizeDes = FinalSysSizing(CurSysNum).DesCoolVolFlow;
+                                } else if (SELECT_CASE_var == Heating) {
+                                    AutosizeDes = FinalSysSizing(CurSysNum).DesHeatVolFlow;
+                                } else if (SELECT_CASE_var == Other) {
+                                    AutosizeDes = FinalSysSizing(CurSysNum).DesMainVolFlow;
+                                } else {
+                                    AutosizeDes = FinalSysSizing(CurSysNum).DesMainVolFlow;
+                                }
+                            }
+                            if (SizeHRHXtoMinFlow) {
+                                if (FinalSysSizing(CurSysNum).DesOutAirVolFlow > 0.0) {
+                                    AutosizeDes = FinalSysSizing(CurSysNum).DesOutAirVolFlow;
```

Comment:
The HR heat exchanger nominal flow rate is autosized to maximum OA flow if the SizeHRHXtoMinFlow is false and the heat exchanger is on OA system.

Reply:
This change does not affect HR heat exchanger nominal flow sizing if the HR unit is on air loop main branch.